src/forms/Textarea.jsx

```diff
@@ -128,6 +128,7 @@ export class Textarea extends React.PureComponent<Props, State> {
 			required,
 			requiredText,
 			disableResize,
+			placeholder,
```

Comment:
Make sure to pass `placeholder` through to `textarea` since it is being destructured from other